net/xtables-wgobfs/Makefile

```diff
@@ -0,0 +1,89 @@
+#
+# Copyright (C) 2022 Wei Chen <weichen302@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=xtables-wgobfs
+PKG_VERSION:=0.4.2
+PKG_RELEASE:=1
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/xt_wgobfs-$(PKG_VERSION)
+PKG_SOURCE:=xt_wgobfs-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:= https://github.com/infinet/xt_wgobfs/releases/download/v$(PKG_VERSION)/
+PKG_HASH:=09fa493d8305e1fa3224a940cab607b1860a9b5d9d395615105c7009e2bec767
+
+PKG_LICENSE:=GPL-2.0
```

Comment:
Do you intend your software to be GPL v2 or later (`GPL-2.0-or-later`) or v2 only (`GPL-2.0-only`)?

Reply:
I am not sure the differences. However, since Jason A. Donenfeld recently confirmed wireguard license is GPL-2.0, it is seems reasonable to use the same license for a kernel module works with WG.

Reply:
> I am not sure the differences.

GPL-2.0-or-later software allows users to apply the terms of GPL 2.0 or 3.0 or any later version to how they use the software. GPL-2.0-only software allows users to apply the terms of GPL 2.0 only. The Linux kernel is GPL-2.0-only.

> However, since Jason A. Donenfeld recently confirmed wireguard license is GPL-2.0, it is seems reasonable to use the same license for a kernel module works with WG.

I'm not sure what the license for WireGuard is exactly, but `GPL-2.0` is the [deprecated](https://spdx.org/licenses/GPL-2.0.html) identifier for `GPL-2.0-only`.

Reply:
Thank you for the explanation! I have update it to `GPL-2.0-only`.